db.json

```diff
@@ -37,7 +37,7 @@
                         "intro_en_us": "He runs gigantic business",
                         "intro_zh_hant": "馬誼郎，出生於南非的美國企業家",
                         "website_url": "https://docs.djangoproject.com/en/3.1/ref/models/expressions/",
-                        "logo_url": "https://i.imgur.com/RK2bZI7.jpeg"
+                        "logo_url": "https://storage.googleapis.com/pycontw-static/sponsors/kloudless/Frame_183.png?Expires=1650879478&GoogleAccessId=web-storage%40pycontw-225217.iam.gserviceaccount.com&Signature=L2UCoan8HhMYlrr%2B9tdDgQQLND3l%2BWrQ7JxqmMGGDwUq5KHRry3BKZypQZNjNGFDxgK2DBNjWynNGCDHwsFpV%2FhzFPRg2tWwAxnbxNrL58n7Axm0GKA4N%2BkUyPlIh%2Fc%2BZD9yROhUw3rWBWaZGb8RKnMxtg%2FTVP0fXDPnst5xMJDAnvxFTU1fOupCgRIxhtuajtvGG%2BMRVq36gya3pf3zmCx2%2BLOdwqR9n94MDq3Q1ASR2Thb%2Fa9OA1MJp%2BTly9QcW0%2FhZXYEgfNTwHGX%2FFHvazxrHKzxjuBIRJQGHDprQmrs3MH1eTDxrOUqo07zn63bhW7jQGGPPCbDhNzr9G%2BlRQ%3D%3D"
```

Comment:
I'm ok with replacing it with real sponsor logos, but it's actually a signed URL of the GCS object and will be expired tomorrow. If you really want to use the logo here, I think the alternatives are:
- Generate another signed URL but with no expiration time
- Upload to any other free picture storage platform

Reply:
Oops, thanks for the reminder. I've rollbacked it to the original one.